services/rds/parameter_group.go

```diff
@@ -13,25 +13,85 @@ import (
 	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
 )
 
-var (
-	pGroupPrefix = pGroupPrefixReal
-)
-
-// PgroupPrefix is the prefix for all pgroups created by the broker.
-const pGroupPrefixReal = "cg-aws-broker-"
 const pgCronLibraryName = "pg_cron"
 
 type parameterGroupAdapterInterface interface {
-	provisionCustomParameterGroupIfNecessary(
-		i *RDSInstance,
-		d *dedicatedDBAdapter,
-		svc rdsiface.RDSAPI,
-	) (string, error)
+	ProvisionCustomParameterGroupIfNecessary(i *RDSInstance) error
+}
+
+type parameterGroupAdapter struct {
+	rds                  rdsiface.RDSAPI
+	settings             config.Settings
+	parameterGroupPrefix string
+}
+
+type paramDetails struct {
+	value       string
+	applyMethod string
+}
+
+func NewParameterGroupAdapater(rds rdsiface.RDSAPI, settings config.Settings) *parameterGroupAdapter {
```

Comment:
```suggestion
func NewParameterGroupAdapter(rds rdsiface.RDSAPI, settings config.Settings) *parameterGroupAdapter {
```

Last change from me 😄

Reply:
2081aceacc39f3de08b7ac3a92eacd06ab9a51d9